src/test/java/lambda/part3/exercise/ReachIterableTest.java

```diff
@@ -25,16 +28,14 @@
         default ReachIterable<T> filter(final Predicate<T> predicate) {
             return consumer -> {
                 boolean hasNext = true;
-                final boolean[] matched = { false };
+                final AtomicBoolean matched = new AtomicBoolean(false);
```

Comment:
I can't understand why, but last commit does not shown here!o_O
But able [here](https://github.com/Leo-Scream/lambda/blob/reach-iterator/src/test/java/lambda/part3/exercise/ReachIterableTest.java)